staging/src/k8s.io/legacy-cloud-providers/azure/azure_vmss.go

```diff
@@ -219,21 +223,21 @@ func (ss *scaleSet) getVmssVMByInstanceID(resourceGroup, scaleSetName, instanceI
 		return vm, found, nil
 	}
 
-	vm, found, err := getter()
+	vm, found, err := getter(crt)
 	if err != nil {
 		return nil, err
 	}
-	if found {
-		return vm, nil
+	if !found {
+		klog.V(3).Infof("Couldn't find VMSS VM with scaleSetName %q and instanceID %q, refreshing the cache", scaleSetName, instanceID)
```

Comment:
I would suggest set as `klog.V(2).Infof`, so that we could check how many ForceRefresh in controller-manager easilly

Reply:
@andyzhangx yes, I'll update that in the new PR.